### PR TITLE
Rename Eth1.0 and Eth2.0

### LINKED ISSUE #333: Eth2 renaming
See this article: https://decrypt.co/91149/ethereum-foundation-kills-eth-2-consensus-layer-rebrand

We need to be consistent on our wording with other consensus layer clients. Once complete, perhaps ask Ben Edgington to review.

### PR DESCRIPTION
**Pull request by rolandtyler**
## Describe the change
Update the Eth1 to "execution layer" and Eth2 to "consensus layer" per the [Ethereum Foundation update](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/). 

## Issue fixed
Fixes #333 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

## Screenshots / recording
https://pegasys-teku--341.com.readthedocs.build/en/341/

### CODE REVIEW

**Comment by benjaminion** [comment]
Well this is [awkward](https://twitter.com/benjaminion_xyz/status/1489864897475264513).

I'd like to make a few changes - I'll add a commit in a little while.

**Review by benjaminion — APPROVED** [review]
Looks good. I've made some updates and corrections in a few places - please check they render OK before merging, I haven't done that.

Otherwise, good to go.

File: docs/HowTo/Get-Started/Connect/Connect-To-Mainnet.md [hunk 1]
```diff
@@ -4,39 +4,44 @@ description: How to connect to Mainnet
 
 # Connect to Mainnet
 
-The following instructions provide the steps run validators on the Ethereum 2.0 Mainnet. You can
+!!! note
+
+    This documentation has been updated in line with the name changes [recommended by the Ethereum Foundation](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/).
+    The execution layer is sometimes known as "Ethereum 1.0". The consensus layer is sometimes known as "Ethereum 2.0".
```
Comment (alexandratran) [inline docs/HowTo/Get-Started/Connect/Connect-To-Mainnet.md:10]:
```suggestion
    The execution layer is sometimes known as "Ethereum 1.0." The consensus layer is sometimes known as "Ethereum 2.0."
```

File: docs/HowTo/Get-Started/Connect/Connect-To-Testnet.md [hunk 2]
```diff
@@ -4,7 +4,12 @@ description: How to connect to a testnet
 
 # Connect to a testnet
 
-The following instructions describe the process to connect Teku to an Ethereum 2.0 testnet.
+!!! note
+
+    This documentation has been updated in line with the name changes [recommended by the Ethereum Foundation](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/).
+    The execution layer is sometimes known as "Ethereum 1.0". The consensus layer is sometimes known as "Ethereum 2.0".
```
Comment (alexandratran) [inline docs/HowTo/Get-Started/Connect/Connect-To-Testnet.md:10]:
```suggestion
    The execution layer is sometimes known as "Ethereum 1.0." The consensus layer is sometimes known as "Ethereum 2.0."
```

**Review by alexandratran — APPROVED** [review]
Besides some [punctuation](https://docs.microsoft.com/en-us/style-guide/punctuation/quotation-marks), lgtm.

**Pull request merged**